index-en.html

```diff
@@ -0,0 +1,849 @@
+                <div class="container">
+                    <div class="navbar-header">
+                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                            <span class="sr-only">Toggle navigation</span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                        </button>
+                        <a class="navbar-brand logo" href=''></a>
+                    </div>
+                    <div class="navbar-collapse collapse">
+                        <ul class="nav navbar-nav navbar-right scrollto">
+                            <li><a href="#home">Home</a></li>
+                            <li><a href="#description">About</a></li>
+                            <li><a href="#projects">Projects</a></li>
+                            <li><a href="#events">Events</a></li>
+                            <li><a href="#who">Our team</a></li>
+                            <li><a href="#partners">Sponsors</a></li>
+                            <li><a href="#contact">Join us</a></li>
+                            <li><a href="index.html">Français</a></li>
+                        </ul>
+                    </div><!--/.nav-collapse -->
+                </div><!--/.container -->
+            </div>
+        </section><!--navigation section end here-->
+        <section id="description">
+            <!--about page content-->
+            <div class="about-section padding-80">
+                <div class="container">
+                    <div class="section-heading text-center">
+                        <h4 class="small section-title"><span>Who we are</span></h4>
+                        <h2 class="large section-title">Our team</h2>
+                    </div><!--section heading-->
+                </div><!--container-->
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-duration="0.3s">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-lightbulb-o"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>JDIS Live! </h4>
+                                    <p align="justify">
+                                        JDIS (French for "Jeux et Défis Informatiques de Sherbrooke", literally "Sherbrooke Computer Science Games and Challenges") is a student group at Université de Sherbrooke which is strongly interested in computer science. It is also an AGEG technical group. Whether you are an engineering student or a science student of computer science, or simply a computer science geek, you are more than welcome among us.
```

Comment:
Yes sure but the text is 

> Whether you are an engineering student or a science student of computer science

My proposition is

> Whether you are an engineering student or a computer science student

or could also be

> Whether you are an engineering student or a science student

I should have been clearer